db/migrate/20200310005657_create_subscriptions.rb

```diff
@@ -0,0 +1,11 @@
+class CreateSubscriptions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :subscriptions do |t|
+      t.integer :list_id, null: false
+      t.integer :subscriber_id, null: false
+      t.string :subscriber_type, null: false, default: 'EmailRecipient'
+
```

Comment:
Consider introducing a unique composite index across these 3 columns to mitigate duplicate subscriptions.